.travis.yml

```diff
@@ -22,7 +22,7 @@ addons:
 install:
   - git clone https://github.com/CGAL/cgal.git
   - cd cgal
-  - git checkout releases/CGAL-4.9
+  - git checkout releases/CGAL-4.10
```

Comment:
@xantares already proposed https://github.com/CGAL/cgal-swig-bindings/pull/86, could you check it?